src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -82,6 +82,38 @@ public static IDataView LoadHousingRegressionDataset(MLContext mlContext)
         /// For more details about this dataset, please see https://archive.ics.uci.edu/ml/datasets/adult.
         /// </remarks>
         public static IDataView LoadFeaturizedAdultDataset(MLContext mlContext)
+        {
+            // Create data featurizing pipeline
+            var pipeline = mlContext.Transforms.CopyColumns("Label", "IsOver50K")
+                // Convert categorical features to one-hot vectors
```

Comment:
Do you still need these changes?

Reply:
yes, because the input data for the pipeline is raw data instead of featurizedData, and the onnx model can only be fed with raw data. So I refactor the SamplesDatasetUtils.cs to have a specific function to get raw data